charts/vela-core/templates/test/test-application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  annotations:
+    helm.sh/hook: test-success
+  name: first-vela-app
+spec:
+  components:
+    - name: express-server
+      type: webservice
+      properties:
+        image: crccheck/hello-world
+        port: 8000
+      traits:
+        - type: ingress
+          properties:
+            domain: testsvc.example.com
+            http:
+              "/": 8000
```

Comment:
As the docs said, the test should be a K8s job with command running.  https://helm.sh/docs/topics/chart_tests/

Reply:
> We could add the chart test in `e2e-test` instead of creating new one

Now we detect whtehre there is a change in charts via `ct list-change`, which comares the charts between `master` branch and `PR` branch. If the charts is not changed,  there is no necessary to run `helm install & helm test`. 

Is it necessary to run `helm install & helm test` in `e2e-test`?

> As the docs said, the test should be a K8s job with command running. https://helm.sh/docs/topics/chart_tests/

I will fix it when I am free.

Reply:
> Is it necessary to run helm install & helm test in e2e-test?

I think we can always run the check. Does it cost too much time in CI?

Reply:
> > Is it necessary to run helm install & helm test in e2e-test?
> 
> I think we can always run the check. Does it cost too much time in CI?

I review the charts and it makes sense to run check every time for the image tag of core controller is `latest` in the master branch.

Will adopt to move charts check to `e2e-test`.